plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java

```diff
@@ -316,7 +316,7 @@ public String[] getInnerBlockPrefixes() {
 
     @Override
     public boolean isWordStart(int ch) {
-        return Character.isUnicodeIdentifierStart(ch);
+    	return Character.isUnicodeIdentifierStart(ch) || ch == 95;
```

Comment:
```suggestion
        return Character.isUnicodeIdentifierStart(ch) || ch == '_';
```

Reply:
Hi, @E1izabeth, I have made the change, thanks for the suggestion!